dask_sql/environment.yml

```
@@ -13,4 +13,5 @@ dependencies:
 - numpy>=NUMPY_VER
 - ucx-proc=*=gpu
 - ucx-py=UCX_PY_VER
-- xgboost=*rapidsaiRAPIDS_VER
+- xgboost=*=rapidsai_*
+- libxgboost=*=rapidsai_*
```

Would suggest the following tweak for a closer match

```suggestion
- xgboost=*=rapidsai_h*
- libxgboost=*=rapidsai_py*
```

---

Sure, that makes sense:

```suggestion
- xgboost=*=rapidsai_py*
- libxgboost=*=rapidsai_h*
```